src/test/java/core/basesyntax/WrongInputTests.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.handler.impl.BalanceHandler;
+import core.basesyntax.handler.impl.PurchaseHandler;
+import core.basesyntax.handler.impl.ReturnHandler;
+import core.basesyntax.handler.impl.SupplyHandler;
+import core.basesyntax.models.FruitTransaction;
+import core.basesyntax.servises.FruitService;
+import core.basesyntax.servises.impl.FruitServiceImpl;
+import core.basesyntax.strategy.Strategy;
+import core.basesyntax.strategy.impl.StrategyImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WrongInputTests {

```

Comment:
I can find the corresponding  service in your solution so let’s  move this method  to the corresponding test class